mobile-app/app/screens/AppNavigator/screens/Loans/screens/PaybackLoanScreen.tsx

```diff
@@ -75,15 +75,18 @@ export function PaybackLoanScreen ({
     const id = tokenId === '0' ? '0_unified' : tokenId
     return new BigNumber(tokens.find((t) => t.id === id)?.amount ?? 0)
   }
+  const getAvailableLoanAmountToPay = (): string => {
```

Comment:
Alignment looks wonky. Instead of using `>`, maybe you can use https://mikemcl.github.io/bignumber.js/#max?

Reply:
Validate against `tokenBalance` because `loanTokenAmount.amount` is not in USD